packages/web-api/src/providers/scan-data-provider.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
can you move these files to service-library to be consistent. (Some data providers may be used from batch as well)

Reply:
no, we do not have this case right now, this is only needed for scan-batch func only, when we will get the case we can definitely refactor it out later.